### ISSUE TITLE
Insert_table with excel like formatting

### ISSUE THREAD

**Issue by rpeys** [scanny/python-pptx#294]
![image](https://cloud.githubusercontent.com/assets/19732554/26785096/c19446b2-49ce-11e7-804a-56762a02471e.png)

Here is an example of a slide containing the kind of table I want:
![image](https://cloud.githubusercontent.com/assets/19732554/26785024/7a57c1b6-49ce-11e7-975d-faab9341546f.png)

**Comment by scanny** [comment]
Well, I think the approach I would take is to see what it takes to get the table looking the way you want using the PowerPoint UI, starting from a default table it inserts. This would be a series of selections and menu commands, like change column width, change cell margins, etc.

Then I'd see which of those are supported by the `Table` methods in `python-pptx` and see which ones weren't yet.

Then I'd develop a workaround function to provide that missing functionality with low-level `lxml` calls operating on the closest XML element I could get from `python-pptx` (e.g. `p:tbl` or `p:tc`).

That would give the flexibility to generate tables having an arbitrary number of rows and columns.

If that looked like more work than I was willing to do I'd go with the "replace the data in an existing table" approach, although that has its limitations as well, requiring you to know the number of tables you want ahead of time and have those "template" slides built into the starting deck.

I don't think approach 4. would work without saving the Excel as an image by hand using the Excel UI.

You're right about approach 5., items on the slide layout appear as static background items on the slide and no shape object for that item appears in the `slide.shapes` collection for the slide.

**Comment by rpeys** [comment]
Thank you! Your response time is incredible. I will try the approach you described, and will probably follow up soon with questions :)

The "replace the data in an existing table" approach is tempting. I generally have 5 slides in my presentations devoted to these tables, and then a variable number of other slides. If I want the table slides to be in the middle of the presentation, my "template" presentation would also need to have a pre-defined number of blank slides available for the rest of the presentation as well, right? In other words, I won't be able to add new slides before/in between my template table slides, only to the end of the presentation - is that correct? Thanks so much!

**Comment by scanny** [comment]
Yes, that's right. So far anyway, unless you also add workaround code for that relocation/insertion/move operation.
